server/cli/launch.py

```diff
@@ -113,6 +113,17 @@ def launch(
     else:
         warnings.formatwarning = custom_format_warning
 
+    if scripts:
+        click.echo("""
+/ / /\ \ \__ _ _ __ _ __ (_)_ __   __ _
+\ \/  \/ / _` | '__| '_ \| | '_ \ / _` |
+ \  /\  / (_| | |  | | | | | | | | (_| |
+  \/  \/ \__,_|_|  |_| |_|_|_| |_|\__, |
+                                  |___/
+The --scripts flag is intended for developers to include google analytics etc. You could be opening yourself to a
+security risk by including the --scripts flag. Make sure you trust the scripts that you are including.
+        """)
```

Comment:
I would further make people type in a confirmation that they understand this message before spawning the browser.

Reply:
We have users who want to deploy in a docker-deployed environment, ie, scripted startup where they have control over the deployment script.   I like Tony's suggestion, but we need to ensure that the confirmation can be scripting (eg, pipe `yes` to it, or whatever).    Alternative to "|yes": provide a `--I-attest-that-really-mean-to-inject-scripts` CLI param that still prints the warning, but dispenses with the interactive prompt.    @ttung - thoughts?

Reply:
I should be able to add this such that `| yes` works.

Reply:
Either way is fine.  The browser being fired up is sufficient, I believe, for bad stuff to run, so I do think the confirmation is important, at least for the first time.

Reply:
Added confirm.

`yes | cellxgene launch --scripts foo --scripts bar example-dataset/pbmc3k.h5ad` bypasses the confirm as expected.